<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgRadialCutter</class>
 <widget class="QDialog" name="dlgRadialCutter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>510</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMargin">
   <property name="geometry">
    <rect>
     <x>684</x>
     <y>140</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>44</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Layers:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_points">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>100</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFeedSpeed">
   <property name="geometry">
    <rect>
     <x>684</x>
     <y>60</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_SafeZ">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>180</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblUseline">
   <property name="geometry">
    <rect>
     <x>554</x>
     <y>190</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>nd Line</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lines">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>60</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_stepdown">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>140</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>190</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>use every</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbrect">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>20</y>
     <width>105</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Rectangular</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>594</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>feed speed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>70</y>
     <width>136</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Stepover (dLine2Line):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>190</y>
     <width>44</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Safe Z</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>230</y>
     <width>491</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="cbRotate">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRotateStep">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>135</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRotateDegrees">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>360</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Degrees Step by</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>20</y>
     <width>59</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Direction:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLeaveMargin">
   <property name="geometry">
    <rect>
     <x>574</x>
     <y>150</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>leave margin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_resolution">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>100</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>36</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Lines:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_xdir">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>20</y>
     <width>36</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;X</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_layers">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>140</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePlungeSpeed">
   <property name="geometry">
    <rect>
     <x>684</x>
     <y>100</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>230</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>smooth</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_stepover">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>60</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ydir">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>20</y>
     <width>37</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Y</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditStatus">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>340</y>
     <width>491</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUseLine">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>180</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSmooth">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>220</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>110</y>
     <width>69</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Resolution:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>584</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>plunge speed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>150</y>
     <width>65</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Stepdown:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>42</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Points:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgRadialCutter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgRadialCutter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
